src/PathOrderOptimizer.h

```diff
@@ -501,7 +501,19 @@ class PathOrderOptimizer
                     score += 1000; //1 meter penalty.
                 }
             }
-            if(score < best_score)
+
+            constexpr float EPSILON = 25.0;
+            if(best_score - EPSILON <= score && score <= best_score + EPSILON)
+            {
+                // add breaker for two candidate starting location with similar score
+                // if we don't do this then we (can) get an un-even seam
+                Point tie_breaker_point = target_pos != Point(0, 0)
+                    ? Point(0, 0)       // break ties by picking the point that is closest to the origin
+                    : Point(0, 100000); // unless the target position is the origin, then pick some other location
+                best_point = vSize2(best_point - tie_breaker_point) < vSize2(best_point - tie_breaker_point) ? best_point : here;
```

Comment:
`vSize2(best_point - tie_breaker_point) < vSize2(best_point - tie_breaker_point)`

is always false I think? Did you mean to put something different on one side of this statement?

Reply:
oops, should be
```suggestion
                 best_point = vSize2(best_point - tie_breaker_point) < vSize2(here - tie_breaker_point) ? best_point : here; 
```

Reply:
https://github.com/Ultimaker/CuraEngine/commit/4e137f5273716dedea5678f32183a601bc71fb6e